src/Tizen.NUI/src/internal/Disposable.cs

```diff
@@ -181,5 +186,14 @@ internal bool SwigCMemOwn
         {
             get => swigCMemOwn;
         }
+
+        /// <summary>
+        /// A Flag to check if it is already disposed.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected bool Disposed
```

Comment:
I've modified to internal protected

Reply:
@huiyueun 
I think you need to add `[EditorBrowsable(EditorBrowsableState.Never)]` for `internal protected`. because derived calss can access `internal protected` types same as `protected` types.

Reply:
@dasg34 ok, Thank you.